sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -604,6 +604,7 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product {
     }.toList
   }
 
+  // TODO: Fix toJSON so that we can more safely handle Map and Seq with loop.
```

Comment:
I removed this comments. I tried, but it seems it requires a big block to explain what this TODO mean. I feel it may creates bigger confusion.